lib/lita/handlers/github_pinger.rb

```diff
@@ -212,6 +222,14 @@ def send_dm(username, content)
           alert_eng_pr("Could not find user with name #{username}, please configure everbot.")
         end
       end
+
+      def detect_type(body)
+        if body['pull_request']
+          'pull_request'
+        elsif body['issue']
+          'issue'
```

Comment:
Was contemplating between having the `_` earlier, and removing them later, or vice versa. Ended up keeping the `_` here as it this could possibly be used as a hash key in more places. No strong opinions